docs/packs/packs-format.md

```diff
@@ -154,16 +154,26 @@ This file will be used while running the `demisto-sdk secrets`([explanation](htt
 **Note**: We use `demisto-sdk secrets` as part of our pre-commit hook to check that possible secrets in the PR aren't exposed to a public repository.
 
 ### .pack-ignore
-This file allows ignoring linter errors while lint checking and ignoring tests in the test collection.
+1) This file allows ignoring linter errors while lint checking and ignoring tests in the test collection.
 
-To add ignored tests/linter errors in a file, first, add the file name to the **.pack-ignore** in this format
+   To add ignored tests/linter errors in a file, first, add the file name to the **.pack-ignore** in this format
 ```
 [file:integration-to-ignore.yml]
 ```
 
 On the following line add `ignore=` flag, with one or more comma-separated values:
-1. `auto-test` - ignore test file in the build test collection.
-2. `linter code` e.g. IN126 - ignore linter error codes.
+* `auto-test` - ignore test file in the build test collection.
+* `linter code` e.g. IN126 - ignore linter error codes.
+
+2) By default, unit-tests of scripts/integrations are running without docker network.
+
+   In case one of the integrations/scripts inside a pack needs network during unit-tests run, this can be done in this format
```

Comment:
```suggestion
   In case one of the integrations/scripts inside a pack needs a network during the unit-tests run, this can be done in this format
```